katdal/visdatav4.py

```diff
@@ -301,6 +347,26 @@ def _before(date):
         _add_sensor_alias(self.sensor, 'Antennas/array/activity', 'obs_activity')
         _add_sensor_alias(self.sensor, 'Antennas/array/target', 'cbf_target')
 
+        # ------ Extract correlator settings ------
+
+        self.sensor['Correlator/instrument'] = CategoricalData(
+            [self._cbf_instrument], all_dumps)
+        try:
+            sync_time = attrs[self._cbf_instrument + '_sync_time']
+        except KeyError:
+            pass
+        else:
+            self.sensor['Correlator/sync_time'] = CategoricalData(
+                [sync_time], all_dumps)
+        try:
+            scale_factor_timestamp = attrs[self._cbf_instrument
+                                           + '_scale_factor_timestamp']
+        except KeyError:
+            pass
+        else:
+            self.sensor['Correlator/scale_factor_timestamp'] = CategoricalData(
+               [scale_factor_timestamp], all_dumps)
+
```

Comment:
Seems a bit inconsistent that `Correlator/instrument` always exists but might be UNKNOWN while these other sensors will no exist at all if the information is not present.